.github/CODEOWNERS

```diff
@@ -1 +1,3 @@
-* @lezzago @qreshi @skkosuri-amzn @bowenlan-amzn @rishabhmaurya @tlfeng @getsaurabh02
\ No newline at end of file
+# This should match the owning team set up in https://github.com/orgs/opensearch-project/teams @opensearch-project/common-utils
+@lezzago @qreshi @bowenlan-amzn @rishabhmaurya @tlfeng @getsaurabh02 @eirsep @sbcd90 @AWSHurneyt
```

Comment:
Just realize you remove the prefix `*` before the 1st user.
Please open another PR and add back.

Thanks.

Reply:
@lezzago @getsaurabh02 @eirsep ^^

Reply:
https://github.com/opensearch-project/common-utils/pull/440